### ISSUE TITLE
Bug: VLC - lyric:(null)

### ISSUE THREAD

**Issue by sassmann** [osdlyrics/osdlyrics#60]
Trying to get osdlyrics working on fedora 29 with vlc. Compiled 0.5.5-rc1.

When I start osdlyrics I get a black semi-transparent window. When starting a new audio track in vlc there are no lyrics displayed.
osdlyrics seems to pick up some info from vlc when the track changes, see log below, but there's debug output saying "lyric:(null)". Any help is appreciated!

log: https://paste.fedoraproject.org/paste/dutqiR0u7ArkeMgggj1dIA/raw

Interesting part of the log:
*Info*: in function _track_changed_cb: ol_main.c[327]
_track_changed_cb
*Info*: in function ol_osd_module_set_lrc: ol_osd_module.c[685]
ol_osd_module_set_lrc
*Info*: in function hide_lyrics: ol_osd_module.c[785]
hide_lyrics
*Debug*: in function ol_osd_window_set_lyric: ol_osd_window.c[1313]
  lyric:(null)
*Info*: in function ol_osd_window_update_lyric_rect: ol_osd_window.c[1292]
ol_osd_window_update_lyric_rect
*Debug*: in function ol_osd_window_set_lyric: ol_osd_window.c[1313]
  lyric:(null)
*Info*: in function ol_osd_window_update_lyric_rect: ol_osd_window.c[1292]
ol_osd_window_update_lyric_rect

**Comment by xaengceilbiths** [comment]
I caught the same(?) bug long long ago. Maybe VLC doesn't send the correct time/location information when playing a WAV file with CUE. So now I'm using deadbeef.

**Comment by sassmann** [comment]
Thanks for the pointer to deadbeef. I tested it, but the debug output still shows the "lyric:(null)".

**Comment by PedroHLC** [comment]
@sassmann you're having the same result for both VLC ans deadbeef?

**Comment by xaengceilbiths** [comment]
Let me provide more information (to determine whether we are talking about the same thing).
I tested an album with CDImage.cue for VLC and CDImage.m3u8 for deadbeef.
The second song is longer than the first one. When I use deadbeef, everything works fine. When I use VLC to play the second song, the lyric starts in the middle. **However, when the song goes to end, there is no lyric:(null) in debug message.**

Would you please provide the lyric file? Maybe an encoding problem?

**Comment by sassmann** [comment]
@PedroHLC sorry my bad. The file I tried didn't proper ID3 tag. After adding the tags the lyrics started showing up. 
However I noticed the following in both VLC and deadbeef. After adding a few tracks to the playlist the first track playing shows lyrics fine, but when press the "next track" button in about 80% of the time only the first lyric line shows up and never starts playing.
Pressing Stop and Play and the lyrics will start playing normally.
